Dockerfile

```
@@ -91,6 +91,17 @@ RUN mkdir -p ${GOPATH}/src/github.com/TheThingsIndustries/protoc-gen-gogottn &&
     go build -ldflags '-w -s' -o /protoc-gen-gogottn-out/protoc-gen-gogottn . && \
     install -Ds /protoc-gen-gogottn-out/protoc-gen-gogottn /out/usr/bin/protoc-gen-gogottn
 
+ARG PROTOC_GEN_GQL_VERSION
+RUN mkdir -p ${GOPATH}/src/github.com/danielvladco/go-proto-gql && \
+    curl -sSL https://api.github.com/repos/danielvladco/go-proto-gql/tarball/v${PROTOC_GEN_GQL_VERSION} | tar xz --strip 1 -C ${GOPATH}/src/github.com/danielvladco/go-proto-gql && \
+    cd ${GOPATH}/src/github.com/danielvladco/go-proto-gql && \
+    go install ./protoc-gen-gql && \
```

Please do the same as for other Go binaries:
```suggestion
    go build -ldflags '-w -s' -o /protoc-gen-gql-out/protoc-gen-gql ./protoc-gen-gql && \
    install -Ds /protoc-gen-gql-out/protoc-gen-gql /out/usr/bin/protoc-gen-gql && \
```

See https://github.com/TheThingsIndustries/docker-protobuf/pull/41/files#diff-3254677a7917c6c01f55212f86c57fbfR117-R118 for example